docs/DeveloperGuide.md

```diff
@@ -419,6 +419,21 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Select an activity to display its details**
+
+**MSS**
+
+1. User select an activity by its index
+2. HobbyList show all the details of the selected activity, including name, description, tags, date, status, rating, review
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The given index is invalid
```

Comment:
Should be 1a. HobbyList detects that the index is invalid.
order is 1 -> 1a..., not 1 -> 2a.